styleguide/components/callouts/callouts.stories.mdx

```diff
@@ -8,9 +8,11 @@ import { translate } from '../../../scripts/story-helpers';
 
 # Callouts
 
-There are several different variants of the callout component:
+There are 4 main callout styles in our Design system:
```

Comment:
Nit: Should this be Design System or design system (capitalisation)?

Reply:
Love this kind of detail, hyphen or dash? :grimacing: 
Let's have a rule and stick to it. Should we use Sentence Case for any specific names, such as Design System, Related Content, and Targeted Content?

Reply:
Yeah I'd use the title case version like Design System for describing the system as a whole (just capitalising Design doesn't make sense), sounds sensible to do that for components too/

Reply:
Actually, I'll just update "Design System" for now. As this is just a draft and will require a content designer (probably Adam) to review, I think he is the best person to look at these details.